client/doc/onboarding.md

```diff
@@ -61,6 +61,30 @@ Go directly to the given step and trigger required events. Useful for go back in
 Returns a step by its name.
 
 ### Step
+
+#### isActive(user)
+* `user`: JS Object
+
+Returns true if the step has to be active for the given `user`. Returns `true` by default.
+This method can be overriden by specifying an `isActive` method in constructor parameter.
+
+##### Example
+```javascript
+let step = new Step({
```

Comment:
Could you precize where should be this code? Where is it called on this project? Is it Application.initialize()?